index.html

```diff
@@ -88,9 +89,9 @@ <h1>Fall Limited Edition Sneakers</h1>
              -->
             <div>
                 <div>
-                    <button type="button" onclick="" onkeydown="">+</button>
-                    <p aria-label="0">0</p>
-                    <button type="button" onclick="" onkeydown="">-</button>
+                    <button type="button"  id ="plus" onclick=" addToCart()" onkeydown="">+</button>
+                    <p aria-label="0" id="text"> 0 </p>
+                    <button type="button"  id ="minus" onclick="removeFromCart()" onkeydown="">-</button>
```

Comment:
onClick & onkeydown calls same methods but using different events. So do add method in keyDown